src/pybind/mgr/cephadm/serve.py

```diff
@@ -1297,6 +1298,7 @@ async def _create_daemon(self,
                             extra_entrypoint_args=ArgumentSpec.map_json(
                                 extra_entrypoint_args,
                             ),
+                            init_containers=init_containers,
```

Comment:
do we also want to set these up like `extra_container_args` and `extra_entrypoint_args` where if they're modified we'll redeploy the daemon automatically?

Reply:
Good point thanks, I'll look into it.

Reply:
We discussed this in the weekly orch meeting and agreed to add it in a follow up PR.